cases/era5_2013.yaml

```diff
@@ -1,19 +1,30 @@
-# Data paths
-land_sea_mask: ../era5_2013/FloodCase_201305_lsm.nc
-region:
+# Settings needed for the data preprocessing (used by preprocess_XXXXXX.py)
 input_folder: ../era5_2013
-interdata_folder: ../output_data
-name_of_run: ""
+preproces_start_date: '20130521' #YYYYMMDD
+preproces_end_date: '20130604' #YYYYMMDD
 
-# Time
-start_date: '20130521'
-end_date: '20130604'
-
-# divt & count_time
+#######################
+# divt & count_time # TODO: Peter
 timestep: 3600  # s; ERA5 data comes at hourly time steps
 divt: 60  # division of the timestep, 96 means a calculation timestep of 24/96 = 0.25 hours (numerical stability purposes)
+########################
+
+# Settings shared by preprocessing and backtracking
+preprocessed_data_folder: ../preprocessed_data
+
+# Settings needed to define the tracking region (in space and time)
+region: ../era5_2013/FloodCase_201305_lsm.nc # requirements: a) same extent in space as preprocessed data, c) values between 0 and 1
+track_start_date: '20130521' #YYYYMMDD
+track_end_date: '20130604' #YYYYMMDD
+
+# Settings needed for the tracking run
+name_of_run: 'default_run'
+veryfirstrun: true # true: starts from zero tracked water, false: loads tracked water
+Kvf: 3 # Vertical transport parameter for gross vertical transport between the layers during the tracking: "actual exchange = Kvf * F_vertical + F_vertical" in one direction and "-1 * (Kvf * F_vertical)" in opposite direction. # Default = 3.
```

Comment:
This comment is quite elaborate. Someday/maybe it might fit better in the documentation, with a short note/pointer in the inline code

Reply:
could indeed be part of readme, but not priority now